lib/stat_tracker.rb

```diff
@@ -36,13 +36,13 @@ def lowest_total_score
 #------------TeamStatistics
 
   def team_info
-    result = {}
+    result = { }
     teams.each do |team|
-      result[:team_id] = team.team_id
-      result[:franchise_id] = team.franchise_id
-      result[:team_name] = team.team_name
-      result[:abbreviation] = team.abbreviation
-      result[:link] = team.link
+      (result[:team_id]||=[]) << team.team_id
```

Comment:
Great use of ||= to continue adding on elements!

Reply:
I agree with @Stearnzy! Good job!